newsroom/history.py

```diff
@@ -49,15 +49,15 @@ def transform(item):
                 continue
 
 
-def get_history_users(item_ids, users, companies):
+def get_history_users(item_ids, active_users, active_companies):
 
     lookup = {'item': {'$in': item_ids}}
 
     histories = query_resource('history', lookup=lookup, max_results=200)
 
     user_matches = []
     for history in histories:
-        if str(history.get('user', '')) in users and str(history.get('company', '')) in companies:
+        if str(history.get('user', '')) in active_users and str(history.get('company', '')) in active_companies:
```

Comment:
wouldn't it be better to have the lookup like: `{'item': {'$in': item_ids}, 'user': {'$in': active_users}}` ?